### PR TITLE
Enable official build

### PR DESCRIPTION
**Pull request by alespergl**

### CODE REVIEW

**Comment by alespergl** [comment]
Yes, this is meant to produce working Electron with `is_official_build = true`. I will request review/merge once ready. There will be a corresponding PR for Electron.

**Comment by alexeykuzmin** [comment]
@alespergl To resolve size related issues we can use `split_static_library` and `split_count`, e.g. here 5ec5f967ce0981a9a6af18cd23a9fe4c55441026.

**Comment by alespergl** [comment]
True, I forgot about that option. Thanks for the tip! 👍

File: chromiumcontent/build_libs.py [hunk 1]
```diff
@@ -275,6 +274,13 @@ def gen_list(out, name, obj_dirs):
             "third_party/WebKit/Source/web",
         ])
 
+    gen_list(
+        out,
+        "obj_webkitcore",
+        [
+            "third_party/WebKit/Source/core",
```
[the remark this answers is not in the document]
Comment (alespergl) [reply chromiumcontent/build_libs.py]: I'd prefer not to use `split_static_library` if not necessary. The problem with it is that it splits only the list of input files with no regard to their size, so it doesn't really solve the size problem unless the inputs are more or less the same size.

[the remark this answers is not in the document]
Comment (alexeykuzmin) [reply chromiumcontent/build_libs.py]: Splitting libs manually doesn't seem to be a good solution, it requires too much work.
If we end up with the same number of libraries, `split_static_library` looks better for me.

**Comment by alespergl** [comment]
@jkleinsc Do you know why the Jenkins job is failing in such a weird way?

**Comment by ckerr** [comment]
@alespergl, @jkleinsc is out sick today so there may be a response lag

**Comment by jkleinsc** [comment]
@alespergl I was trying something in Jenkins that required a plug-in that wasn’t installed.  I installed the plugin and I’ll rerun the failed job

**Comment by alespergl** [comment]
@jkleinsc If you want to restart the arm64 build, that's fine. But the same code built all green before, so it shouldn't be necessary.

Otherwise this is ready to be merged, preferably not squashed. Then I will update https://github.com/electron/electron/pull/11847 to the new libcc master.

**Pull request merged**